src/main/scala/com/gd/twitteranalytics/JsonTranslator.scala

```diff
@@ -0,0 +1,23 @@
+package com.gd.twitteranalytics
+
+import org.apache.log4j.Logger
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object JsonTranslator {
+  val log = Logger.getLogger(JsonTranslator.getClass.getName)
+
+  def getParsedJsonAsDataFrame(spark:SparkSession,path:String):DataFrame ={
+    import spark.implicits._
+
+    val unParsedDf = spark.read.parquet(path)
+    log.debug("=======> Stage 1: Twitter Data Read Start..")
```

Comment:
log messages that specify order of an operation in a workflow  ("Stage #....") should be in a "flow controller" logic, not inside of a component.

Reply:
Fixed..